docs/extensions-plugins/cli/commands.mdx

```diff
@@ -72,13 +72,23 @@ Lists all accessible AI models.
 pieces list models
 ```
 
-### `open`
-Opens an asset from a list or search. If only "open" is used, it opens the most recent asset.
+#### open (deprecated)
+Opens an asset from a list or search. If only "open" is used, it opens the most recent asset. This also creates a link to the asset's code.
 
 ```bash
-pieces open [ITEM_INDEX]
+pieces open [ITEM_INDEX] [-e]
 ```
 
+```-e``` is an optional flag . It Opens the asset in the configured external editor. Editor of choice can be configured using config command.
+
+ ##### Editor Configuration:
+
+ You can configure an external editor to open assets for editing. Use the following command to set your preferred editor:
+
+ ```pieces config editor <editor_command>```
+
+#### NOTE : previous implementations of the open command are now deprecated. 
```

Comment:
Can we use either of the following rather than having `NOTE:`

[Info Admonition](https://docusaurus.io/docs/markdown-features/admonitions). We use these all across the docs and its a nice way to show some extra info about something.

or 

Just have the link be this: `> Previous implementations of the open command are now deprecated.`

Reply:
used the second suggestion in commit [94e8b7a](https://github.com/pieces-app/documentation/pull/538/commits/94e8b7a438416dd239894a9ddf5a322f00343c6d)